src/status_im/ui/components/styles.cljs

```diff
@@ -109,8 +109,11 @@
    :height 24})
 
 (def icon-add
-  {:width  14
-   :height 14})
+  {:width 14
+   :height 14
+   :color color-blue4
```

Comment:
We recently introduced a `colors` ns. Please switch to it.

Reply:
Also please align the values in the map

Reply:
Also please align the values in the map